Syllabus.md

```diff
@@ -0,0 +1,115 @@
+# GCB/CIS/MTR 535: Introduction to Bioinformatics
+
+## Course directors
+
+### Benjamin F. Voight, PhD
+Department of Systems Pharmacology and Translational Therapeutics
+Department of Genetics
+10-126 Smilow Center for Translational Research (SCTR)
+3400 Civic Center Blvd
+215-746-8083
+bvoight@upenn.edu
+
+### Casey S. Greene, PhD
+Department of Systems Pharmacology and Translational Therapeutics
+10-131 Smilow Center for Translational Research (SCTR)
+3400 Civic Center Blvd
+215-573-2991
+csgreene@upenn.edu
+
+## Teaching assistants
+
+* [Katie Siewert](mailto:ksiewert@mail.med.upenn.edu)
+* TBD
+
+### Guest proctors:
+
+* [Greg Grant](mailto:ggrant543@gmail.com)
+
+## Location and time
+
+TBD
+
+## Office Hours
+
+BFV: Wednesday 11-12
+CSG: Friday 11-12
+TBD
+
+## Course web Site
+
+We will use [SageMathCloud](https://cloud.sagemath.com/) for in class exercises
+and homework. Completed assignments will be collected through this system.
```

Comment:
Grades will be returned via canvas? Will make this what it says and then let @ksiewert / TAs decide what the best way to return this is.